server/src/server_host.ts

```diff
@@ -65,6 +65,9 @@ export class ServerHost implements ts.server.ServerHost {
   }
 
   fileExists(path: string): boolean {
+    if (path.endsWith('.ngtypecheck.ts')) {
```

Comment:
Can you add a quick comment here noting why this is necessary? Without the context of the PR, it would be difficult to know why there's a special case in this function for typecheck files.

Reply:
comment added